core/controllers/classifier.py

```diff
@@ -123,6 +125,24 @@ def post(self):
             classifier_services.get_classifier_training_job_by_id(job_id))
         if classifier_training_job.status == (
                 feconf.TRAINING_JOB_STATUS_FAILED):
+            # Send email to admin and admin specified email recipients.
+            # Other email recipients are specified on admin config page.
+            mail_subject = 'Failed ML Job %s' % job_id
+            admin_mail_body = ((
+                'ML job %s has failed. For more information,'
+                'please visit the admin page.') % job_id)
+            email_manager.send_mail_to_admin(mail_subject, admin_mail_body)
+            other_recipients = (
+                email_manager.NOTIFICATION_EMAILS_FOR_FAILED_TASKS.value)
+            if other_recipients:
+                other_mail_body = ((
+                    'ML job %s has failed. For more information,'
```

Comment:
It's fine for this to be the same instructions as for the admin case above. In general the people added to the email field would be admins anyway.